Documentation build action failing

**Describe the bug**

Our doc build is currently failing with the following error:
```
Extension error (sphinx_gallery.docs_resolv):
Handler <function embed_code_links at 0x7ff2b571e7a0> for event 'build-finished' threw an exception (exception: Expecting property name enclosed in double quotes: line 1 column 2 (char 1))
Traceback (most recent call last):
  File "/usr/share/miniconda/envs/docs/bin/sphinx-multiversion", line 8, in <module>
    sys.exit(main())
  File "/usr/share/miniconda/envs/docs/lib/python3.7/site-packages/sphinx_multiversion/main.py", line 338, in main
    subprocess.check_call(cmd, cwd=current_cwd)
  File "/usr/share/miniconda/envs/docs/lib/python3.7/subprocess.py", line 363, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '('/usr/share/miniconda/envs/docs/bin/python', '-R', '-m', 'sphinx', '-D', 'smv_metadata_path=/tmp/tmpafqyjjva/versions.json', '-D', 'smv_current_version=main', '-c', '/home/runner/work/librosa/librosa/docs', '/tmp/tmpafqyjjva/a19da5075663321d49afd9beca574e4ee527f5b7/docs', '/home/runner/work/librosa/librosa/build/html/main')' returned non-zero exit status 2.
Error: Process completed with exit code 1.
```

This was observed after merging #1487 , but I'm pretty sure it's due to the may 30 release of sphinx 5.0.  (Docs build clean on my dev machine with sphinx 4.5.0.)  I'm not seeing any obvious culprits in the 5.0 [release notes](https://www.sphinx-doc.org/en/master/changes.html#release-5-0-0-released-may-30-2022).

For now, I'll pin the version to <5 in the doc build environment, and we can revisit later on.

--- a/.github/environment-docs.yml
+++ b/.github/environment-docs.yml
@@ -19,7 +19,7 @@ dependencies:
   - pooch>=1.0.0
   - packaging>=20.0
   - matplotlib<3.5
-  - sphinx>=4.4.0
+  - sphinx==4.5.0
   - sphinx-gallery>=0.10.1
   - sphinx_rtd_theme>=1.0.0
   - ffmpeg

